includes/class-sensei-admin.php

```diff
@@ -79,7 +79,6 @@ public function __construct() {
 		// Add AJAX endpoint for event logging.
 		add_action( 'wp_ajax_sensei_log_event', array( $this, 'ajax_log_event' ) );
 
-		Sensei_Home::instance()->init();
```

Comment:
Moving this to main Sensei class because it needs to be initialized in the REST API as well.